weblate/README.md

```diff
@@ -223,31 +223,31 @@ for i in `find libo_ui-master -name [^.]\*.po`; do sed -i -f fixerrors.sed $i; d
 Check the changes by means of the `potrans.py -p ui diff` command. If something went wrong revert the changes (`potrans.pu -p ui re`), modify the sed file and run `sed` again.
 
 #### 4. Upload the modifications
-Upload the modifications by `potrans.py -p ui up` to the weblate server.
+Upload the modifications by `potrans.py -p ui up` to the Weblate server.
 
 ## Translate strings using Google translate
 Translation services as Google translate make sense only for longer text strings composed of sentences. GUI translation should be done directly in Weblate.
 ### 1. Export the desired substrings
-Use the `export` command (eventually with one of its switches). If necessary, sort the csv file according to text length and delete rows short text.
+Use the `export` command (eventually with one of its switches). If necessary, sort the CSV file according to text length and delete rows short on text.
 ### 2. Use the translation service
-Using LibreOffice copy some rows in the Source column and translate them using https://translate.google.com/. Copy the translation to the Target column.
+Using LibreOffice, copy some rows in the Source column and translate them using https://translate.google.com/. Copy the translation to the Target column.
```

Comment:
I thought this was done in Weblate, which the earlier change about machine-translation is about.